### PR TITLE
[Docs] Consolidated Compatibility Matrix on Supported Platforms page

### LINKED ISSUE #2515: [Docs] Compatibility Matrix
**Current State:**
Integrations are a key aspect of the value Meshery offers to users. Details of the compatibility of Meshery and its components are available in various places in Meshery Docs (in Jekyll datafiles). Bringing this information together into a single location in a compatibility matrix (maybe multiple tables) will benefit users.

**Desired State:**
Create a visual compatibility matrix showing Meshery and `mesheryctl`'s compatibility with various systems. Include the following information:

- Operating System (if available - reuse existing information)
- Kuberenetes platform (reuse existing information)
- Service meshes (reuse existing information)

Two tables may be necessary to show supported platforms for Meshery and for `mesheryctl`.

**Examples**
Screen Shot 2021-03-09 at 7 39 49 AM

Link to [VMware compatibility matrix](https://interopmatrix.vmware.com/#/Interoperability?isHideGenSupported=true&isHideTechSupported=true&isHideCompatible=false&isHideIncompatible=false&isHideNTCompatible=false&isHideNotSupported=true&isCollection=false&col=2,5088,4276,3496,3457,3222,2862,2736,3364,2732,2332,2132,2136,998,695,431,792,611,578,508,1034,442,444,448,254,1035,462,392,252,140,2981,194,172,38,242,173,37,36,35&row=0,)

#### Child Issues

- [ ] [CI] Centralize Meshery Adapter Integration Test Results #4741 

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

**Description**
image

This PR fixes #2515 #6280 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_includes/compatibilityMatrix.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+{% endfor %}
+{% elsif k8s_item.name == "meshery-traefik-mesh" %}
+{% assign traefik_size = k8s_item.size | times:1.0 %}
+{% for single in k8s_item.items %}
+{% if single.overall-status == "passing" %}
+{% assign successfull_traefik_mesh = successfull_traefik_mesh | plus:1 %}
+{% endif %}
+{% endfor %}
+{% endif %}
+{% endfor %}
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply docs/_includes/compatibilityMatrix.md:142]: More accurately, we didn't have any before. 😬

[the remark this answers is not in the document]
Comment (asubedy) [reply docs/_includes/compatibilityMatrix.md:142]: Yes, 😬

**Comment by l5io** [comment]
🚀 Preview for commit ae7083890081b7dbfc0bf711c4e74911bd083649 at: https://636bdd2f94e7f907793a95b4--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 90ffdf11df4544c21584057b38ec675156b0efa9 at: https://636bff2067454804a3e923f6--meshery-docs-preview.netlify.app

File: docs/_includes/compatibilityMatrix.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (asubedy) [reply docs/_includes/compatibilityMatrix.md]: Fixed

**Comment by l5io** [comment]
🚀 Preview for commit e56c55f3ee814cdffc55398f7cb165d1d9189037 at: https://636c0aa5b9f8130efb789557--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@asubedy we have a few of places in different docs or different sites that point to the https://docs.meshery.io/project/compatibility-matrix page. A logical step here is to decommission that page and have it `redirect_to` the consolidated, new compatibility matrix (or for that page to `redirect_from`).

**Comment by leecalcote** [comment]
@asubedy updated?

**Comment by l5io** [comment]
🚀 Preview for commit 9491222036ea77d17817bb3517aad4206ca7f2ea at: https://6372395edbbc1037c44fb860--meshery-docs-preview.netlify.app

**Comment by asubedy** [comment]
Updated..

**Comment by leecalcote** [comment]
Windows - https://6372395edbbc1037c44fb860--meshery-docs-preview.netlify.app/installation/platforms/windows
Docker and Mac compatibility can be included.

**Comment by asubedy** [comment]
image
This on the main table right?

**Comment by leecalcote** [comment]
Right - include that on the main table. We need something else other than the blank chart.

**Comment by asubedy** [comment]
image

**Comment by l5io** [comment]
🚀 Preview for commit 3bbc5a8a787f603a21fabdac12bda5f0ea5db751 at: https://637403dc05752a280a8c7a1a--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
That looks great, @asubedy. Now for Docker. ;)

**Comment by l5io** [comment]
🚀 Preview for commit 8d767e111328df88cf22ee06a2cb23bbb4ee29cd at: https://6374d7314db6c7058657a9d4--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@asubedy a little less than two hours before deadline.... Let's just remove Mac and Windows. 

Under this new design, where can visitors see the overview of Integration Test Status? https://docs.meshery.io/project/compatibility-matrix#integration-tests

**Comment by asubedy** [comment]
Ok removing windows and Mac, 
image
A similar kind of information section can be added just below the Compatibility matrix that links to the integration test

**Comment by asubedy** [comment]
Windows and Mac removed and added a info box under Kubernetes section that links to integration tests page

**Comment by l5io** [comment]
🚀 Preview for commit 1699c380475325877d66ee2002ee8e8df5d8c930 at: https://6374f2d945d7231b948dd97b--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
💥 "Looks perfect" according to the 20+ people on today's meeting.

**Pull request merged**